src/payment/strategies/sage-pay/sage-pay-payment-strategy.ts

```diff
@@ -22,18 +25,39 @@ export default class SagePayPaymentStrategy extends CreditCardPaymentStrategy {
     }
 
     execute(payload: OrderRequestBody, options?: PaymentRequestOptions): Promise<InternalCheckoutSelectors> {
+        if (!payload.payment) {
+            throw new NotInitializedError(NotInitializedErrorType.PaymentNotInitialized);
+        }
+
+        if (this._isThreeDSTwoExperimentOn()) {
+            payload.payment.paymentData = {
```

Comment:
Also where is this used?

Reply:
It is used here `return super.execute(payload, options)` so all the browser_info is sent to [bigpay-client-js ](https://github.com/bigcommerce/bigpay-client-js/pull/125/files#diff-3da05841578808d50f2a80c3653813d5686ef321688400f603b71809929e26b3R77) and then its sent to bigpay